x-pack/legacy/plugins/integrations_manager/public/screens/detail/index.tsx

```diff
@@ -46,13 +46,14 @@ export function DetailLayout(props: LayoutProps) {
     padding-bottom: ${theme.eui.paddingSizes.xl};
   `;
 
-  const FullWidthRemainingHeight = styled(EuiPage)`
+  const paddingSizeTop: number = parseInt(theme.eui.paddingSizes.xl, 10) * 1.25;
```

Comment:
We now have 3 of these "`xl` times some multiple" values. Doesn't have to be fixed here, but I want to keep an eye out for them and make some EPM theme values if we need to repeat them.